CHANGELOG.md

```diff
@@ -22,16 +22,17 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - Removed some duplicated strings
   - Removed some old unused strings
   - Fixed some broken source strings (line names)
-- Simplified Chinese and Traditional Chinese localization updates (by @sbzlzh)
+- Simplified Chinese and Traditional Chinese localization updates (by @sbzlzh):
   - Update Simplified Chinese Translation
-  - Improve translation(by @TheOnly8Z)
+  - Improve translation (by @TheOnly8Z)
 
 ### Fixed
 
 - Fixed hotreload of TTT2 roles library by a fresh reinitialization
 - Fixed a wrong localization line call in roles.lua (by @Satton2)
 - Fixed +zoom bind
 - Fixed ttt_quickslot command
+- Fixed the error message in entspaw.lua and weapon_ttt_spawneditor.lua when the player is not respawned in the map tttt_submarine, and the inability to generate weapons and customize weapons under this map. And the language selection when pressing F1s suddenly shows an error. (by @sbzlzh)
```

Comment:
This seems a bit to convoluted. Maybe the following would work better:

```suggestion
- Fixed an issue in `table.GetEqualEntryKeys` when nil is provided instead of a table. (by @sbzlzh)
  - This fixes spawn problems on maps with invalid spawn points
  - This fixes errors in the F1 Menu language selection
```